examples/valijson_nlohmann_bundled.hpp

```diff
@@ -83,11 +83,11 @@ VALIJSON_NORETURN inline void throwNotSupported() {
 #
 # if defined __clang_major__
 #   if (__clang_major__ == 3 && __clang_minor__ >= 5)
-#     define TR2_OPTIONAL_CLANG_3_5_AND_HIGHTER_
+#     define TR2_OPTIONAL_CLANG_3_5_AND_HIGHER_
```

Comment:
I'm hoping that this being an example doesn't have any consumers who care about the change...

It's possible this is an from an upstream. I'm happy to send it upstream (this PR is an upstream from winget-cli...)

Reply:
Happy to accept these. While this has been corrected upstream, there are some modifications in this file to allow exceptions to be disabled across Valijson. It's very unlikely it will ever be updated with any further upstream changes.